tinygrad/codegen/uops.py

```diff
@@ -148,14 +148,15 @@ def print(self):
       print(f"{self.uops.index(u):4d} {str(u.uop):20s}: {str(u.dtype) if u.dtype is not None else '':25s} "
             f"{str([self.uops.index(x) for x in u.vin]):32s} {u.arg}")
 
-  def add(self, uop:UOps, dtype:Optional[DType]=None, vin:Tuple[UOp, ...]=tuple(), arg:Any=None, cachable=True, insert_before=None,
-          simplify=True) -> UOp:
+  def add(self, uop:UOps, dtype:Optional[DType]=None, vin:Tuple[UOp, ...]=tuple(), arg:Any=None, insert_before:Optional[int|UOp]=None,
```

Comment:
Let's keep this interface as it was - that comment was just for `self.const`. I think insert_before in uops.add should be kept as positional